docs/sources/auth/generic-oauth.md

```diff
@@ -55,6 +55,8 @@ You can also specify the SSL/TLS configuration used by the client.
 
 Set `empty_scopes` to true to use an empty scope during authentication. By default, Grafana uses `user:email` as scope.
 
+### E-mail address
```

Comment:
```suggestion
### Email address
```

Reply:
I was just following the used convention.  I personally don't mind "e-mail" as a clearer indication of the name's origin, and here in the doc as a delineation between the meaning and the config.
Would rather an e-mail spelling change separate since this isn't the subject of this PR and makes it less clear.

Reply:
Our [style guide[(https://github.com/grafana/grafana/blob/main/contribute/style-guides/documentation-style-guide.md) says that you should follow the Google developer documentation style guide, which uses email and not e-mail.